mlos_bench/mlos_bench/tests/environments/composite_env_test.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Unit tests for composite environment.
+"""
+
+import pytest
+
+from mlos_bench.environments.composite_env import CompositeEnv
+from mlos_bench.tunables.tunable_groups import TunableGroups
+from mlos_bench.services.config_persistence import ConfigPersistenceService
+
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture
+def composite_env(tunable_groups: TunableGroups) -> CompositeEnv:
+    """
+    Test fixture for CompositeEnv.
+    """
+    return CompositeEnv(
+        name="Composite Test Environment",
+        config={
```

Comment:
Another example that included nested CompositeEnvironments would also be good, but our config schema doesn't currently support that directly (have to use `include_children`), so maybe we fix both in a future PR.

Reply:
To do that we can use a variant on [recursive json schemas](http://json-schema.org/understanding-json-schema/structuring.html#recursion).